src/lib/lightning/eclair/eclairService.ts

```diff
@@ -81,10 +81,11 @@ class EclairService implements LightningService {
           uniqueId: c.channelId.slice(-12),
           channelPoint: c.channelId,
           pubkey: c.nodeId,
-          capacity: commitInput.amountSatoshis,
+          capacity: commitInput.amountSatoshis.toString(10),
```

Comment:
Previously we were passing `capacity` as a number here when we expect a `string`.

Reply:
Good catch!